src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -560,6 +566,67 @@ internal sealed class SortedSetEntryArrayProcessor : ValuePairInterleavedProcess
                 new SortedSetEntry(first.AsRedisValue(), second.TryGetDouble(out double val) ? val : double.NaN);
         }
 
+        internal sealed class SortedSetSpanProcessor : ResultProcessor<SortedSetSpan?>
+        {
+            protected override bool SetResultCore(PhysicalConnection connection, Message message, in RawResult result)
+            {
+                if(result.Type == ResultType.MultiBulk)
+                {
+                    if (result.IsNull)
+                    {
+                        return true;
```

Comment:
This returns a violation of the API, should set the `<typename>.Null` here (see global comments).